pkg/reconciler/instances/ory/test/ory_integration_test.go

```diff
@@ -0,0 +1,63 @@
+package test
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/kyma-incubator/reconciler/pkg/logger"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/kubernetes"
+	"github.com/kyma-incubator/reconciler/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubectl/pkg/util/podutils"
+)
+
+const (
+	envOryIntegrationTests = "ORY_RECONCILER_INTEGRATION_TESTS"
+	namespace              = "kyma-system"
+)
+
+func OryIntegrationTest(t *testing.T) {
+	if !isIntegrationTestEnabled() {
+		t.Skipf("Integration tests disabled: skipping parts of test case '%s'", t.Name())
+	}
+}
+
+func isIntegrationTestEnabled() bool {
+	expensiveTests, ok := os.LookupEnv(envOryIntegrationTests)
+	return ok && (expensiveTests == "1" || strings.ToLower(expensiveTests) == "true")
```

Comment:
Why do you mix conditions? You verbosely set the env value to `1` so it is what should be expected. Envs are strings so they will not be converted into boolean out-of-the-box.

Reply:
Makes sense, we should refactor this code from reconciler to our use case.